monkey/infection_monkey/utils/timer.py

```diff
@@ -10,13 +11,13 @@ def __init__(self):
         self._timeout_sec = 0
         self._start_time = 0
 
-    def set(self, timeout_sec: float):
+    def set(self, timeout_sec: float, start_time: Optional[float] = None):
         """
         Set a timer
         :param float timeout_sec: A fractional number of seconds to set the timeout for.
```

Comment:
change comment

Reply:
```suggestion
        :param float timeout_sec:  A nonnegative floating point number expressing seconds
```

Reply:
No, I meant that we need to add `start_time` as well